package.json

```diff
@@ -145,54 +153,55 @@
     ]
   },
   "dependencies": {
-    "use-sync-external-store": "1.1.0"
+    "date-fns": "2.28.0",
+    "use-sync-external-store": "1.2.0"
```

Comment:
Can we not update dependencies in this PR? `date-fns` shouldn't be here.

Reply:
Some `devDependencies` need to be updated so that they recognise `out` annotation and the build is successful, I wasn't sure which ones in particular are needed so I updated all of them. If you know which in particular are required I can update only them.

And yeah we don't need to update `dependencies` and not sure how `date-fns` ended up there, I'll revert this bit.